controllers/storagecluster/placement.go

```diff
@@ -38,6 +38,12 @@ func getPlacement(sc *ocsv1.StorageCluster, component string) rookCephv1.Placeme
 		return placement
 	}
 
+	// if provider-server placements are found in the storagecluster spec append the default ocs tolerations to it
+	if ok && component == "provider-server" {
```

Comment:
is there any reason that we don't want to set default tolerations if they are not defined in the spec?

Reply:
We are setting default ones already at line no. 21.

Reply:
no you are not, line 21 is hit if `ok == false` and line 43 is hit if `ok == true` they are mutually exclusive isn't it or am I still missing something?

Reply:
`ok == false` means nothing is defined by the user in the storagecluster CR and in that case we are returning the default one as expected.